ludwig/schema/metadata/configs/trainer.yaml

```diff
@@ -70,6 +70,15 @@ ecd:
             In many large-scale training runs, evaluation is often configured to run on
             a sub-epoch time scale, or every few thousand steps.
         ui_display_name: Checkpoints per epoch
+    layers_to_freeze_regex:
+        default_value_reasoning:
+            Default no layers will be frozen for fine-tuning a pretrained model.
```

Comment:
what is the repercussion of not freezing any layers

Reply:
No repercussion. If no layers are frozen, normal full fine tuning (adjusting all parameters) will take place.